src/isIsogram.test.js

```diff
@@ -6,4 +6,30 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return 'false' if argument is not a string`, () => {
+    expect(isIsogram(42)).toBe(false);
+  });
+
+  it(`should return 'true' if argument is empty string`, () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it(`should return 'true' for a single character 'a'`, () => {
+    expect(isIsogram('a')).toBe(true);
+    expect(isIsogram('Z')).toBe(true);
+  });
+
+  it(`should be case-insensitive 'M' or 'm' return 'true'`, () => {
+    expect(isIsogram('CaSe')).toBe(true);
+  });
```

Comment:
The description of this test case is a bit misleading. It suggests that 'M' or 'm' should return 'true', but the test case is actually checking if the function is case-insensitive by using the word 'CaSe'. Consider rephrasing the description to better reflect the test case.